phac-nml/irida-next
# File import UI enhancements

## What does this PR do and why?
Added the `Import metadata` button & dialog to the group samples page.

This is an extension of https://github.com/phac-nml/irida-next/pull/714.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/5cfe6385-4aa0-4901-b455-479f022fb7e0)
![image](https://github.com/user-attachments/assets/67dafa87-7570-45b1-af67-40c485c3627c)

## How to set up and validate locally
1. Navigate to a group samples page that the logged in user has access to update.
2. Hit the `Import Metadata` button.
3. Select a file. (Can be a modified line list export.)
4. Select the `Sample ID Column`.
5. Hit the `Import Metadata` button.
6. Verify the sample metadata was updated.
7. Try with and without ignoring empty values.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
